sockeye/transformer.py

```diff
@@ -226,13 +217,12 @@ def hybrid_forward(self, F,
                        source: mx.sym.Symbol,
                        source_bias: mx.sym.Symbol,
                        autoregr_states: mx.sym.Symbol,
-                       enc_att_k: Optional[mx.sym.Symbol] = None,
-                       enc_att_v: Optional[mx.sym.Symbol] = None) -> Tuple[mx.sym.Symbol,
+                       enc_att_kv: Optional[mx.sym.Symbol] = None) -> Tuple[mx.sym.Symbol,
                                                                            mx.sym.Symbol]:
-        target_autoregr, *new_autoregr_states = self.autoregr_layer(self.pre_autoregr_layer(target, None),
```

Comment:
As mentioned above, we should maintain the unpacking in order to keep supporting autoregressive layers with more than one state tensor.